tracing-subscriber/src/lib.rs

```diff
@@ -10,8 +10,71 @@
 //! `tracing-subscriber` is intended for use by both `Subscriber` authors and
 //! application authors using `tracing` to instrument their applications.
 //!
+//! ## Usage
+//!
+//! First, add this to your `Cargo.toml` file:
+//!
+//! ```toml
+//! [dependencies]
+//! tracing-subscriber = "0.1"
+//! ```
+//!
+//! Add the following to your executable to initialize the default subscriber:
```

Comment:
This should probably in the `fmt` module's docs. This crate might be used for other reasons as well.

Reply:
Are you okay keeping the docs for this in the README, or would you prefer to take those out and just point to `docs.rs`?

Reply:
What do you think about removing it from the `subscriber` crate's readme, but putting it in the repo's _root_ readme? It needs some more basic getting started-type docs (see #384).

Reply:
Sure thing. Want me to just remove it from this PR and submit a follow-up with examples for the root crate?

Reply:
I'm fine with adding it to the root readme now, or in a follow up, whatever you prefer!